carvis-web/src/server/routes/redisRoutes.js

```diff
@@ -0,0 +1,95 @@
+/* ======== REDIS SCHEMA =====
+places:
+---------
+hash name: place (string)
+keys:
+---
+userId:lat : int
+userId:lng : int
+userId:routable : string
+
+-- otherwise do getOwnPropertyNames to check another userId:data
+// NOTE: or is there a better way?
+
+===============================
+userdata:
+---------
+hash name: userId (alexa userId)
+keys:
+---
+lyftToken: string
+lyftPaymentInfo: string
+uberToken: string
+originlat: int
+originlng: int
+originRoutable: string
+phoneNumber: int
+...
+
+===============================
+
+*/
+
+
+var bluebird = require("bluebird");
+bluebird.promisifyAll(require('redis')); // promisified versions have 'Async' postfix - ie. client.getAsync() and client.hgetallAsync()
+var client = redis.createClient();
+
+// this sets the Redis server as an LRU cache with 500MB space.
+client.config("SET", "maxmemory", "500mb"); // should this be bigger ?
```

Comment:
i would actually err on the side of small at 1st and only increase as and when we need to, to limit $ etc